docs/doc/10-deploy/02-deploying-with-cloud-object-storage.md

```diff
@@ -0,0 +1,134 @@
+---
+title: Deploying with Cloud Object Storage
+sidebar_label: Deploying with Cloud Object Storage
+description:
+  How to deploy Databend with cloud object storage
+---
+
+## Deploying with Cloud Object Storage
+
+Databend works with both self-hosted and cloud object storage solutions. This topic explains how to deploy Databend with cloud object storage. For a list of compatible cloud object storage solutions, see [Understanding Deployment Modes](./00-understanding-deployment-modes.md). 
+
+In this topic, we will deploy a standalone Databend on a local host, then connect it to your cloud object storage.
+
+### Setting up Cloud Object Storage
+
+Before deploying Databend, make sure you have successfully set up your object storage environment in the cloud, and the following tasks have been completed:
+
+- Create a bucket or container named *databend*.
+- Get the endpoint URL for connecting to the bucket or container you created.
+- Get the Access Key ID and Secret Access Key for your account.
+
+:::tip
+
+**Need Help?** For information about how to manage buckets and Access Keys for your cloud object storage, refer to the user manual from the solution provider. Here are some useful links you may need:
```

Comment:
I believe that most users do not know how to configure `endpoint_url`, here we must clearly point it out for the different object storage.
This is why we list so many cloud storages(oss/qingstore/cos/s3) to the document.

Reply:
Agreed.